flepimop/gempyor_pkg/src/gempyor/compartments.py

```diff
@@ -600,10 +610,10 @@ def parse_parameter_strings_to_numpy_arrays(
             not operators
         ):  # empty list means all have been tried. Usually there just remains one string in string_list at that time.
             raise ValueError(
-                f"""Could not parse string {string_list}. 
-    This usually mean that '{string_list[0]}' is a parameter name that is not defined
-    or that it contains an operator that is not in the list of supported operator: ^,*,/,+,-.
-    The defined parameters are {parameter_names}."""
+                f"Could not parse string '{string_list}'. " 
+                f"This usually mean that '{string_list[0]}' is a parameter name that is not defined "
+                f"or that it contains an operator that is not in the list of supported operators: ^,*,/,+,-. "
```

Comment:
supported operators is defined somewhere else, yes? let's use that lookup object somehow to create the supported list in this error.

Reply:
good note thank you